app/src/androidTest/assets/features/create_profile.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Create a profile
+  When you create a profile
+
+  Scenario: Clicking submit with nothing filled in
+    Given I am viewing the create profile activity
+    When I click on the submit profile button
+    And I click ok on the popup
+    Then I should stay on the create profile activity
+    And the name input field should become red
+    And the password input field should become red
+    And the repeat password input field should become red
+
+  Scenario Outline: Clicking submit with valid input
+      Given I am viewing the create profile activity
+      And there are no profiles in the database
+      When I enter a valid <name> as name
+      And I enter a valid <password> as password
+      And I enter a valid repeat <password> as repeat password
+      And I click on the submit profile button
+      And I click ok on the popup
```

Comment:
Can you specify what kind of popup this is? This will make it easier to read the feature and prevent duplicate `When`'s